tests/Api/Responses/Expected/shop/get_taxonomy_response.json

```diff
@@ -9,49 +9,17 @@
             "id": "@integer@",
             "code": "MUG",
             "children": [],
-            "translations": {
-                "de_DE": {
-                    "@id": "\/api\/v2\/shop\/taxon-translations\/@integer@",
-                    "@type": "TaxonTranslation",
-                    "id": "@integer@",
-                    "name": "Tassen",
-                    "slug": "tassen",
-                    "description": "Einige Beschreibung Lorem ipsum dolor sit amet."
-                },
-                "en_US": {
-                    "@id": "\/api\/v2\/shop\/taxon-translations\/@integer@",
-                    "@type": "TaxonTranslation",
-                    "id": "@integer@",
-                    "name": "Mugs",
-                    "slug": "categories\/mugs",
-                    "description": "Some description Lorem ipsum dolor sit amet."
-                }
-            }
+            "name": "Mugs",
+            "description": "Some description Lorem ipsum dolor sit amet."
```

Comment:
Lets add slug to the response (or did you had reason not to?). Also, please mention about this change in UPGRADE-API-1.11.md